src/app/feedback/feedback.component.ts

```diff
@@ -95,12 +96,14 @@ export class FeedbackComponent implements OnInit, AfterViewInit, OnDestroy {
 
   getFeedback() {
     const selector = !this.user.isUserAdmin ? { 'owner': this.user.name } : { '_id': { '$gt': null } };
-    this.couchService.findAll(this.dbName, findDocuments(selector, 0, [ { 'openTime': 'desc' } ]))
-      .subscribe((data) => {
-        this.feedback.data = data;
+    forkJoin([
+      this.couchService.findAll('_users'),
+      this.couchService.findAll(this.dbName, findDocuments(selector, 0, [ { 'openTime': 'desc' } ]))
+    ]).subscribe(([ users, feedbacks ]: [any, any]) => {
+        this.feedback.data = feedbacks.map(feedback => ({ ...feedback, user: users.find(u => u.name === feedback.owner) }));
         this.emptyData = !this.feedback.data.length;
         this.dialogsLoadingService.stop();
-      }, (error) => this.message = 'There is a problem of getting data.');
+    }, (error) => this.message = 'There is a problem of getting data.');   
```

Comment:
The extra spaces should not be allowed by the pre push hook.  Please make sure you have that on your system by running `npm run install-hooks`.